src/ui/public/vis/response_handlers/basic.js

```diff
@@ -74,7 +75,8 @@ const BasicResponseHandlerProvider = function (Private) {
         const tableGroup = aggResponse.tabify(vis.getAggConfig().getResponseAggs(), response, {
           canSplit: true,
           asAggConfigResults: true,
-          isHierarchical: vis.isHierarchical()
+          isHierarchical: vis.isHierarchical(),
+          timeRange: getTime(vis.indexPattern, vis.filters.timeRange).range
```

Comment:
@ppisljar Since we talked earlier about removing `vis.filters`, should we introduce a new reference here? Do we have an idea how we remove that in the future?

Reply:
`vis.filters` is going away in the long run, but the response handlers will still get the access to the `timeRange` (probably being passed in directly as a parameter in the future) so this should not be a problem.

Reply:
@blfrantz If you create a visualization without a time field you will get an error for this line because the `getTime` will return `undefined` if not time field is specified on the selected `indexPattern`.

You can verify the error using the `shakespeare` dataset https://www.elastic.co/guide/en/kibana/current/tutorial-load-dataset.html and just creating a barchart with that index.